packages/handlersjs-http/lib/handlers/routed-http-request.handler.ts

```diff
@@ -30,10 +30,18 @@ export class RoutedHttpRequestHandler extends HttpHandler {
 
     }
 
-    this.pathToRouteMap = new Map(
-      this.handlerControllerList.flatMap((controller) =>
-        controller.routes.map((route) => [ route.path, { controller, route } ])),
-    );
+    this.pathToRouteMap = new Map();
+
+    this.handlerControllerList.flatMap((controller) =>
+      controller.routes.forEach((route) => {
+
+        const existing = this.pathToRouteMap.get(route.path);
+
+        existing
+          ? this.pathToRouteMap.set(route.path, [ ...existing, { controller, route } ])
+          : this.pathToRouteMap.set(route.path, [ { controller, route } ]);
+
+      }));
```

Comment:
I do think this can be cleaner. In fact, I would probably use a `Map<path, Map<method, handler[]>>` construction. But no priority right now.